hyperctl.sh

```diff
@@ -358,7 +370,7 @@ download-image() {
 
   if ! [ -a $IMAGE.$IMGTYPE ]; then
     curl $IMAGEURL/$IMAGE.$IMGTYPE$ARCHIVE -O
-    shasum -a 256 -c <(curl -s $IMAGEURL/$SHA256FILE | grep "$IMAGE.$IMGTYPE$ARCHIVE")
+    shasum -a 256 -c <(curl -s -L $IMAGEURL/$SHA256FILE | grep "$IMAGE.$IMGTYPE$ARCHIVE")
```

Comment:
was this a problem without the `-L` ?

Reply:
The checksums URL responded with a redirect to the actual checksums file.